<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>1304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #fdf6e3;
    font-family: Arial;
    font-size: 14px;
}

QLineEdit {
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 6px;
    background-color: white;
}

QPushButton {
    background-color: #008cba;
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 6px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="logo">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>logoGestionTapas.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Introduce tu email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Introduce tu contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_rol">
       <property name="text">
        <string>Selecciona tu rol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboRol">
       <item>
        <property name="text">
         <string>Cliente</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Empleado</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Administrador</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="text">
        <string>Iniciar sesión</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonRegistro">
       <property name="text">
        <string>Registrarse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonAnonimo">
       <property name="text">
        <string>Entrar como invitado</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelError">
       <property name="styleSheet">
        <string notr="true">color: red;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
